src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -1262,7 +1262,7 @@ export async function showPopupWithGraphic(graphicObject: any, options: any, vie
         let graphic = arcGisObjectRefs[graphicObject.id] as Graphic;
         view.popup.dockOptions = options.dockOptions;
         view.popup.visibleElements = options.visibleElements;
-        view.popup.open({
+        await view.openPopup({
```

Comment:
do we want to adjust the `...popup.open` on line 1245 to follow the `view.popup.open` pattern?

Reply:
nice catch! That one isn't called as often, so I didn't notice it.